runtime/src/runtime/policy.rs

```diff
@@ -206,7 +206,7 @@ impl Default for Policy {
             new_sectors_per_period_max: policy_constants::NEW_SECTORS_PER_PERIOD_MAX,
             chain_finality: policy_constants::CHAIN_FINALITY,
 
-            valid_post_proof_type: HashSet::<RegisteredPoStProof>::from([
+            valid_post_proof_type: BTreeSet::<RegisteredPoStProof>::from([
```

Comment:
Not critical, but it would be _nice_ to implement this as either a bitfield or a vector of booleans. These values are sequential.